A110-child-channel-plugins.md

```diff
@@ -0,0 +1,215 @@
+# **A110: Child Channel Configuration Plugins**
+
+* **Author(s)**: [Abhishek Agrawal](mailto:agrawalabhi@google.com)
```

Comment:
There are a couple of fields missing here, in particular the "approver" and "mailing list link" fields.

Please make sure you are following the gRFC template: https://github.com/grpc/proposal/blob/master/GRFC-TEMPLATE.md